validation/src/main/scala/hmda/validation/rules/lar/validity/V355.scala

```diff
@@ -0,0 +1,25 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object V355 extends EditCheck[LoanApplicationRegister] {
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    val denialReasons = List(lar.denial.reason1, lar.denial.reason2, lar.denial.reason3)
+
+    val validDenialReason = "" :: (1 to 9).map(_.toString()).toList
```

Comment:
I believe that you don't actually have to apply `toList` if you exchange `::` with `++`.